testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -138,8 +138,36 @@ Feature: Smoke tests for <client>
     Then I should see a "1 package install has been scheduled for" text
     And I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
-@suse_minion
-  Scenario: Test Prometheus formula on the <client>
+  Scenario: Test Prometheus exporter formula on the <client>
+    Given I am on the Systems overview page of this "<client>"
+    When I follow "Formulas" in the content area
+    And I check the "prometheus-exporters" formula
+    And I click on "Save"
+    Then I wait until I see "Formula saved" text
+    # Configure Prometheus exporter formula
+    When I follow "Prometheus Exporters" in the content area
+    And I click on "Expand All Sections"
+    And I should see a "Enable and configure Prometheus exporters for managed systems." text
+    And I check "node" exporter
+    And I check "apache" exporter
+    And I check "postgres" exporter
+    And I click on "Save"
+    Then I should see a "Formula saved" text
+    # Apply highstate for Prometheus exporters
+    When I follow "States" in the content area
+    And I click on "Apply Highstate"
+    Then I should see a "Applying the highstate has been scheduled." text
+    And I wait until event "Apply highstate scheduled by admin" is completed
+    # Visit monitoring endpoints on the minion
+    When I wait until "node" exporter service is active on "<client>"
+    And I visit "Prometheus node exporter" endpoint of this "<client>"
+    And I wait until "apache" exporter service is active on "<client>"
+    And I visit "Prometheus apache exporter" endpoint of this "<client>"
+    And I wait until "postgres" exporter service is active on "<client>"
+    And I visit "Prometheus postgres exporter" endpoint of this "<client>"
+
+@monitoring_server
+  Scenario: Test <client> as Monitoring Server using Prometheus formula
```

Comment:
We need to remove completely this scenarion as agreed this morning.

Reply:
this is just being applied on the monitoring server though - not on any other client, hence the tag